src/core/datastore/cachestore.spec.js

```diff
@@ -753,18 +745,12 @@ describe('CacheStore', () => {
         .get(`/appdata/${store.client.appKey}/${collection}`)
         .reply(200, [entity1, entity2]);
 
-      return store.pull()
-        .then(() => {
-          nock(store.client.apiHostname)
-            .delete(`/appdata/${store.client.appKey}/${collection}/${entity1._id}`)
-            .reply(200);
-
-          nock(store.client.apiHostname)
-            .delete(`/appdata/${store.client.appKey}/${collection}/${entity2._id}`)
-            .reply(200);
+      nock(store.client.apiHostname)
+        .delete(`/appdata/${store.client.appKey}/${collection}`)
+        .reply(200);
```

Comment:
Note there is no mocked response - meaning it's totally ignored in the response of the `remove()` method. Only offline operation result is considered.